src/main/java/com/alvarium/annotators/AbstractPkiAnnotator.java

```diff
@@ -5,13 +5,19 @@
 import java.nio.file.Files;
 import java.nio.file.Paths;
 
+import org.apache.logging.log4j.Logger;
+
 import com.alvarium.sign.KeyInfo;
 import com.alvarium.sign.SignException;
 import com.alvarium.sign.SignProvider;
 import com.alvarium.sign.SignProviderFactory;
 import com.alvarium.utils.Encoder;
 
 abstract class AbstractPkiAnnotator extends AbstractAnnotator {
+  AbstractPkiAnnotator(Logger logger) {
+    super(logger);
```

Comment:
I moved setting the logger in the constructor for now and called the super in the custom annotators for now till we check the proposed Logger singleton. But I needed to create a constructor for the AbstractPkiAnnotator to follow the AbstractAnnotator